app/models/recommendation.rb

```diff
@@ -24,4 +24,12 @@ class Recommendation < VersionedRecord
 
   validates :title, presence: true
   validates :reference, presence: true, uniqueness: true
+
+  def is_current
+    reporting_cycle_taxonomy = Taxonomy.joins(:taxonomy).where(has_date: true).where(taxonomies_taxonomies: {priority: 1}).first
```

Comment:
not sure if "date" and "priority" will suffice to identify the "human rights body" taxonomy. we could alternatively set the taxonomy ID as an env variable

Reply:
cool I'll set it as a required environment variable `HUMAN_RIGHTS_BODY_TAXONOMY_ID` so that'll need to be set. Do you want it to default to `1` or would you prefer the app failed to load unless it was set?

Reply:
sure, default to `1` is great, thanks